visualization/app/codeCharta/codeCharta.service.spec.ts

```diff
@@ -343,7 +351,9 @@ describe("codeChartaService", () => {
 				}
 			]
 
-			await codeChartaService.loadFiles([{ fileName, content: validFileContent, fileSize: 42 }])
+			expect(async () => codeChartaService.loadFiles([{ fileName, content: validFileContent, fileSize: 42 }])).rejects.toThrow(
+				"No files could be uploaded"
+			)
```

Comment:
Why can't this file be loaded although it has a valid content?

Reply:
see the test description:

> it("should show a validation error if two files in a folder have the same name"

which is set up above:
```ts
validFileContent.nodes[0].children[1].children[0].name = "duplicate"
validFileContent.nodes[0].children[1].children[1].name = "duplicate"
```